src/libs/ReportUtils.ts

```diff
@@ -1255,16 +1255,16 @@ function getChildReportNotificationPreference(reportAction: OnyxEntry<ReportActi
  * - report is a draft
  * - report is a processing expense report and its policy has Instant reporting frequency
  */
-function canAddTransactionsToMoneyRequest(report: OnyxEntry<Report>): boolean {
-    if (!isIOUReport(report) && !isExpenseReport(report)) {
+function canAddOrDeleteTransactions(moneyRequestReport: OnyxEntry<Report>): boolean {
+    if (!isMoneyRequest(moneyRequestReport)) {
```

Comment:
This might fix the failing tests as well

Reply:
woops just a mistake, will update

Reply:
>if (!isMoneyRequest(moneyRequestReport) && !isMoneyRequestReport(moneyRequestReport)) 

It should only be `isMoneyRequestReport` here, because we can't add transactions to a transaction :D